git-sync/Dockerfile

```diff
@@ -1,4 +1,14 @@
-FROM golang:1.4-onbuild
-VOLUME ["/git"]
+FROM alpine:edge
```

Comment:
I don't think we can use Alpine for containers like this - DNS in Alpine is currently broken (doesn't respect search paths) which means this will not work for anyone who wants to access a git server in your own cluster.